CRM/Upgrade/Incremental/php/FiveFortyThree.php

```diff
@@ -74,6 +74,21 @@ public function upgrade_5_43_alpha1(string $rev): void {
     $this->addTask('Replace membership type token in action schedule',
       'updateActionScheduleToken', 'membership.type', 'membership.membership_type_id:label', $rev
     );
+    $this->addTask('Replace event fee amount token in action schedule',
+      'updateActionScheduleToken', 'event.fee_amount', 'participant.fee_amount', $rev
+    );
+    $this->addTask('Replace event type token in action schedule',
+      'updateActionScheduleToken', 'event.event_type_id', 'participant.event_type_id:label', $rev
```

Comment:
I am not sure why event type should move from event to participant entity as is is a static value that belongs to the event

Reply:
@magnolia61 I think you're right, https://github.com/civicrm/civicrm-core/pull/21738